packages/sdk/src/__tests__/one-file-judge.test.ts

```
@@ -0,0 +1,52 @@
+          repair: false,
+          fallback: false,
+        },
+      },
+    });
+    expect(
+      Workflow.getStopSemantics({
+        mode: "judge",
+        input: { json: "artifacts/submission.json", schema: "artifacts/submission.schema.json" },
+        output: { path: "artifacts/result.json", schema: "artifacts/result.schema.json" },
+        options: { repair: false, fallback: false },
+      }),
```

ESLint

[error] Delete `,`

(prettier/prettier)